src/components/auth/auth-form.vue

```diff
@@ -1,34 +1,75 @@
 <script setup lang="ts">
-import { ref } from 'vue'
+import { onMounted, ref, watch } from 'vue'
 import { useTranslation } from 'i18next-vue'
 import { storeToRefs } from 'pinia'
 
+import * as AuthService from '@/services/auth/auth.service'
+import { useAlertNotificationsStore } from '@/stores/alert-notifications.store'
+import { AlertNotificationType } from '@/stores/alert-notifications.store.model'
 import { useAppStore } from '@/stores/app.store'
+import { useUserManagerStore } from '@/stores/user-manager.store'
+import { User } from '@/stores/user-manager.store.model'
 
 const MYACCOUNT_URL = import.meta.env.VITE_MYACCOUNT_URL
 const MYACCOUNT_RECOVER_URL = import.meta.env.VITE_MYACCOUNT_RECOVER_URL
 const MYACCOUNT_NEW_URL = import.meta.env.VITE_MYACCOUNT_NEW_URL
 
 const { t } = useTranslation()
-const { lang } = storeToRefs(useAppStore())
-const isAuthenticated = ref(false)
-const email = ref('')
+const { addNotification } = useAlertNotificationsStore()
+const { lang, isApp } = storeToRefs(useAppStore())
+const { setCurrentUser, clearUser } = useUserManagerStore()
+const { authenticated, currentUser } = storeToRefs(useUserManagerStore())
 const userName = ref('')
+const userPassword = ref('')
 
-function logout() {
-  alert('TODO: logout')
-  isAuthenticated.value = false
+watch(authenticated, authenticated => {
+  if (authenticated) {
+    addNotification(t('Vous êtes maintenant correctement connecté.'))
+  }
+})
+
+onMounted(() => {
+  AuthService.getUserInfo()
```

Comment:
AuthService.getUserInfo() is called asynchronously here (no await/async) because I don't want to wait for the getInfo() response to render the component.